zlong_alert.zsh

```diff
@@ -29,8 +34,15 @@ zlong_alert_func() {
     local cmd=$1
     local secs=$2
     local ftime=$(printf '%dh:%dm:%ds\n' $(($secs / 3600)) $(($secs % 3600 / 60)) $(($secs % 60)))
-    if [[ "$zlong_use_notify_send" == true ]]; then
-        notify-send "Done: $1" "Time: $ftime"
+    local message="Done: $1 Time: $ftime"
+    if [[ "$zlong_internal_send_notifications" != false ]]; then
+        # Find and use the correct notification command based on OS name
+        if [[ "${uname}" == "Linux" ]]
```

Comment:
looks like ${uname} isn't an env var, but ${OSTYPE} would work

here is a good example 
https://stackoverflow.com/a/8597411

Reply:
`uname` isn't an env var, it's a [preinstalled program](https://man7.org/linux/man-pages/man2/uname.2.html) on Linux.

It worked for me on Ubuntu, so if it doesn't work for everyone then I'm guessing it's not included in all Linux distros by default. Either way, I'm fine switching to `$OSTYPE`. Will update the PR with the change.

Reply:
Have updated it to use $OSTYPE.